sdk/cosmos/tests.springonly.yml

```diff
@@ -0,0 +1,27 @@
+trigger: none
+
+variables:
+  - template: ../../eng/pipelines/templates/variables/globals.yml
+  - name: AdditionalArgs
+    value: ''
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      TestName: 'Integration'
+      MatrixConfigs:
```

Comment:
I'll need you to pull in [these changes](https://github.com/Azure/azure-sdk-for-java/pull/21154/files#diff-3f4f2787651acc85e17527ce2c3a388b5a2ab57d65c018862357c6d630aa420bR42-R46) once that gets merged, since we're moving cosmos tests into their own subscription.